gui/src/main/java/cern/jarrace/inspector/gui/rest/ContainerService.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * © Copyright 2016 CERN. This software is distributed under the terms of the Apache License Version 2.0, copied
+ * verbatim in the file “COPYING”. In applying this licence, CERN does not waive the privileges and immunities granted
+ * to it by virtue of its status as an Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+
+package cern.jarrace.inspector.gui.rest;
+
+import cern.jarrace.commons.domain.AgentContainer;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Path;
+import retrofit2.http.Query;
+
+import java.util.List;
+
+/**
+ * A service that can access a remote endpoint with methods related to {@link AgentContainer}s.
+ */
+public interface ContainerService {
```

Comment:
What I meant is that I completely prefer the way RETROFIT consumes services than Spring and would prefer to stick to retrofit in which case you cannot (you can but its a huge confusion) stick to the same classes for both the client and server for consuming and providing the services. I dunno but then I will change it to use spring on both and pass some classes to the commons package